nextjs-todolist-app/README.md

```diff
@@ -0,0 +1,237 @@
+
+    | **Field**       | **Value**               |
+    |-----------------|-------------------------|
+    | **Name**        | `Todo List Service`  |
+    | **Description** | `Manages Todo lists` |
+
+5. Click **Next**.
+6. To allow Choreo to connect to your GitHub account, click **Authorize with GitHub**.
+7. If you have not already connected your GitHub repository to Choreo, enter your GitHub credentials, and select the repository you created by forking [https://github.com/wso2/choreo-examples](https://github.com/wso2/choreo-examples) to install the [Choreo GitHub App](https://github.com/marketplace/choreo-apps).
+
+    !!! info
+         The **Choreo GitHub App** requires the following permissions:<br/><br/>- Read and write access to code and pull requests.<br/><br/>- Read access to issues and metadata.<br/><br/>You can [revoke access](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-authorized-integrations#reviewing-your-authorized-github-apps) if you do not want Choreo to have access to your GitHub account. However, write access is only needed to send pull requests to a user repository. Choreo will not directly push any changes to a repository.
+
+
+8. In the **Connect Repository** pane, enter the following information:
+
+    | **Field**             | **Description**                               |
+    |-----------------------|-----------------------------------------------|
+    | **GitHub Account**    | Your account                                  |
+    | **GitHub Repository** | **`choreo--samples`** |
+    | **Branch**            | **`main`**                               |
+    | **Buildpack**      | Click **Docker** because you are creating the REST API from a Dockerfile.|
+    | **Dockerfile Path**            | **`nextjs-todolist-app/nexjs-todolist-app-backend/Dockerfile`**                               |
+    | **Docker Context Path**            | **`web-apps/NextJs-App/nexjs-todolist-app-backend`**                               |
+
+9. Click **Create**. This initializes the service with the implementation from your GitHub repository and takes you to the **Overview** page of the component.
+
+###  Step 1.2: Deploy the service
+
+For the REST endpoint of the service to be invokable, you need to deploy it. To deploy the service, follow the steps given below:
+
+1. In the left navigation menu, click **Build** and then select the latest commit and build.
+
+2. In the left navigation menu, click **Deploy** and then select the latest image and build.
+
+2. In the **Setup Area** card, click **Configure & Deploy** from the split button.
+
+!!! info
+
+1.  To successfully build your container with Choreo, it is essential to explicitly define a User ID (UID) under the USER instruction in your Dockerfile. For reference, see [sample Dockerfile](https://github.com/wso2/choreo-sample-apps/blob/main/go/greeter/Dockerfile).
```

Comment:
Can we point to a Docker file in choreo-samples repo?

Reply:
Perhaps this might work https://github.com/wso2/choreo-samples/blob/main/greeting-service-go/Dockerfile